java/code/src/com/suse/manager/webui/utils/LoginHelper.java

```diff
@@ -68,13 +68,12 @@ public class LoginHelper {
 
     private static Logger log = LogManager.getLogger(LoginHelper.class);
     private static final String DEFAULT_KERB_USER_PASSWORD = "0";
-    private static final Long MIN_PG_DB_VERSION = 130001L;
-    private static final Long MAX_PG_DB_VERSION = 149999L;
-    private static final String MIN_PG_DB_VERSION_STRING = "13";
-    private static final String MAX_PG_DB_VERSION_STRING = "14";
-    private static final Double OS_VERSION_CHECK = 15.4;
-    private static final Long OS_VERSION_MIN_DB_VERSION = 140000L;
-    private static final String OS_VERSION_WANTED_DB_VERSION = "14";
+    private static final Long MIN_PG_DB_VERSION = 140001L;
+    private static final Long MAX_PG_DB_VERSION = 150000L;
+    private static final String MIN_PG_DB_VERSION_STRING = "14";
+    private static final String MAX_PG_DB_VERSION_STRING = "16";
+    private static final Double OS_VERSION_CHECK = 15.5;
+    private static final String OS_VERSION_WANTED_DB_VERSION = "16";
```

Comment:
Since check for OS version is being removed in this PR (L376-L381), these two can be removed as well. I grepped the source and did not find any other usage.

```suggestion
```